src/Providers/FeatureLayer.js

```diff
@@ -8,7 +8,7 @@ EsriLeafletGeocoding.Controls.Geosearch.Providers.FeatureLayer = L.esri.Services
     }
   },
   initialize: function(url, options){
-    L.esri.Services.FeatureLayer.prototype.call(this, url, options);
+    L.esri.Services.FeatureLayer.prototype.initialize.call(this, url, options);
```

Comment:
The tests and Travis build are failing on the master because the call function is undefined. It appears to me that it was a typo and 'call' should have been called on the initialize function. I successfully ran the tests and the Travis build is now successful, too.